src/client.ts

```diff
@@ -560,7 +579,7 @@ export class Client extends EventEmitter {
         blockHeight: result.block_height,
         data: B64ToUint8Array(result.encoded_body),
         topics: result.topics,
-        transactionHash: result.tx_hash
+        transactionHash: B64ToUint8Array(result.tx_hash)
```

Comment:
This might break people's code, maybe add a new field `txHashBytes` and leave the old one unchanged?

Reply:
Yeah, I'll add another one for bytes